### ISSUE TITLE
Should users invest a little more effort to "commit" to their Plan?

### ISSUE THREAD

**Issue by abrauninger** [VotingVoices/chorus#64]
I demo'd the app last night to a friend of mine (who works in software too).  After generating her plan, she suggested that we provide a way for users to invest a little more effort to "emotionally commit" to their plan.

For example, one idea is that we not only provide a way to share via social networks, but also that we ask the user to write a personal message talking about their plan.

We may get this "for free" with our social network sharing buttons (Issue #40), but I think it might be worth thinking about other ways to maximize one of the psychological effects that VotePlan is designed to take advantage of: When people think through a plan, they have not only thought about the details, but also they feel an obligation to follow through because someone asked them what their plan was.

In other words, there might be more we could do to increase the (good) sense of obligation that people feel to follow through on their plan.
